themes/squadtheme/static/css/site.css

```diff
@@ -17,8 +17,8 @@ h1 a {
 }
 
 a img {
-    height: 80px;
-    width: 80px;
+    height: 8vw;
+    width: 8vw;
```

Comment:
This is still going to bite us next time we want to add a linked image. We should make this a class specifically for the social icons (I think that's what these are for)

Reply:
Yeah, that's true. I'm okay adding that class later if/when we want to add more images, but can add it now if you want.

Reply:
Yeah I don't want to have to remember this or dig around to find it again when that comes up and the new linked image is for some reason an 8vw square. It's much easier to fix now than later.